src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenuItems-ChangePassword.cshtml

```diff
@@ -0,0 +1,11 @@
+@using OrchardCore.Admin
+@using OrchardCore.Users.Models
+
+@if (!Site.As<LoginSettings>().DisableLocalLogin)
```

Comment:
These checks should be in the drive not the view

Reply:
I thought about moving those instead of duplicate them in the views